tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{py,27}-sa_{1_0,1_1}, py{33,34}-sa_{1_0,1_1}
+envlist = pypy, py27, py33, py34, py35, py36
```

Comment:
please test with bin/tox whether this works: `py{py,27,33,34,35,36}-sa_{1_0,1_1}`